src/core/internal/array/appending.d

```diff
@@ -139,7 +147,7 @@ template _d_arrayappendTImpl(Tarr : T[], T)
  * Run postblit on `t` if it is a struct and needs it.
  * Or if `t` is a array, run it on the children if they have a postblit.
  */
```

Comment:
If the return type is changing, I recommend updating the DDoc comment to include a `Returns` explanation.